OWNERS

```diff
@@ -1,11 +1,8 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/devel/owners.md
-owners:
-  - droot
-  - pwittrock
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
 approvers:
+  - kubebuilder-owners
   - kubebuilder-admins
   - kubebuilder-maintainers
 reviewers:
-  - kubebuilder-admins
-  - kubebuilder-maintainers
```

Comment:
According to https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#maintaining-owners-files, approvers should not be reviewers.